File: Cozinha/Model/Refeicao.cs
namespace Cozinha.Model;

public class Refeicao{
    public long Id { get; set;}
    public required bool IsAtivo { get; set; } = true;
    public required Prato Prato { get; set;}
    public required TipoRefeicao TipoRefeicao { get; set;}
    public int Quantidade { get; set;}
    public DateTime Data{ get; set;}    
}

File: Restaurante/src/app/Models/refeicao.ts
export interface Refeicao {
    id: number;
    prato: {
      id: number;
      nome: string;
      isAtivo: boolean;
      tipoPrato: {
        id: number;
        nome: string;
      };
      ingredientes: {
        id: number;
        nome: string;
        categoriaAlimenticia: string;
        ativo: boolean;
      }[];
      receita: string;
    };
    tipoRefeicao: {
      id: number;
      nome: string;
      descricao: string;
    };
    quantidade: number;
    data: string;
  }
  